### DISCUSSION TITLE
Max joins per second

### DISCUSSION THREAD

**Discussion by H3xtyy** [PurpurMC/Purpur#517] [__Completed__] [answered]
There is a `"max-joins-per-tick:"` in paper.yml, even when you set value to 1, server still can drop tps pretty badly. My feature request is to add in purpur "the same" option but per second. Would help larger servers <3

**Comment by BillyGalbreath** [comment, marked as answer]
Implemented in build 1518

https://github.com/PurpurMC/Purpur/commit/0a9a17044bc4f009dcd3fa8abfed955871212b36

When this purpur.yml option is set to true, the `max-joins-per-tick` setting in paper.yml will be used per second instead of per tick

```yaml
settings:
  network:
    max-joins-per-second: false
```
